<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmImageCalendar</class>
 <widget class="QWidget" name="frmImageCalendar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="ImageCalendar" name="imageCalendar1"/>
   </item>
   <item row="0" column="1">
    <widget class="ImageCalendar" name="imageCalendar2">
     <property name="calendarStyle">
      <enum>ImageCalendar::CalendarStyle_Yellow</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="ImageCalendar" name="imageCalendar3">
     <property name="calendarStyle">
      <enum>ImageCalendar::CalendarStyle_Brown</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="ImageCalendar" name="imageCalendar4">
     <property name="calendarStyle">
      <enum>ImageCalendar::CalendarStyle_Gray</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ImageCalendar" name="imageCalendar5">
     <property name="calendarStyle">
      <enum>ImageCalendar::CalendarStyle_Red</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="ImageCalendar" name="imageCalendar6">
     <property name="calendarStyle">
      <enum>ImageCalendar::CalendarStyle_Purple</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageCalendar</class>
   <extends>QWidget</extends>
   <header>imagecalendar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
